### PR TITLE
Added new updates

### PR DESCRIPTION
**Pull request by monalivadje**

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -33,8 +33,21 @@ jobs:
         # stop the build if there are Python syntax errors or undefined names
         flake8 climt/ --count --show-source --statistics
     - name: Build climt
-      env:
-        CLIMT_ARCH: Linux
       run: |
-        python setup.py build
-
+           if [ "$RUNNER_OS" == "Linux" ]; then
+            CLIMT_ARCH= Linux
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:38]: You cannot leave space during assignment of env variables in bash.
`CLIMT_ARCH= Linux` should be `CLIMT_ARCH=Linux`

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -33,8 +33,21 @@ jobs:
...
+            python setup.py build
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:39]: This is on the same line as the previous assignment
`CLIMT_ARCH=Linux python setup.py build`

File: .github/workflows/python-package.yml [hunk 3]
```diff
@@ -33,8 +33,21 @@ jobs:
...
+           elif [ "$RUNNER_OS"== "Windows"]; then
+            CLIMT_ARCH= Windows
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:41]: Same issue as above

File: .github/workflows/python-package.yml [hunk 4]
```diff
@@ -33,8 +33,21 @@ jobs:
...
+            python setup.py build 
+           elif [ "$RUNNER_OS"== "Darwin"]; then
+            CLIMT_ARCH= Darwin
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:44]: Same issue as above

**Review by JoyMonteiro — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (monalivadje) [reply .github/workflows/python-package.yml]: ok

File: .github/workflows/python-package.yml (hunk 2 above)
Comment (monalivadje) [reply .github/workflows/python-package.yml]: ok

File: .github/workflows/python-package.yml [hunk 5]
```diff
@@ -33,8 +33,21 @@ jobs:
...
       run: |
-        python setup.py build
-
+           if [ "$RUNNER_OS" == "Linux" ]; then
+            CLIMT_ARCH= Linux
+            python setup.py build
+           elif [ "$RUNNER_OS"== "Windows"]; then
+            CLIMT_ARCH= Windows
+            python setup.py build 
+           elif [ "$RUNNER_OS"== "Darwin"]; then
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:41]: I think `$RUNNER_OS` for Mac is `MacOS`, not `Darwin`. Please check

File: .github/workflows/python-package.yml (hunk 4 above)
Comment (monalivadje) [reply .github/workflows/python-package.yml]: done changes

File: .github/workflows/python-package.yml [hunk 6]
```diff
@@ -33,8 +33,21 @@ jobs:
...
+            CLIMT_ARCH= Darwin
+            python setup.py build
+           else 
+             echo "$RUNNER_OS not supported"
+             exit 1
+           fi
+      shell: bash   
+  # - name: Test with pytest
```
Comment (JoyMonteiro) [inline .github/workflows/python-package.yml:48]: You can uncomment the testing also now.

**Review by JoyMonteiro — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package.yml (hunk 5 above)
Comment (monalivadje) [reply .github/workflows/python-package.yml:41]: ok

File: .github/workflows/python-package.yml (hunk 6 above)
Comment (monalivadje) [reply .github/workflows/python-package.yml:48]: ok

**Comment by JoyMonteiro** [comment]
Strange, the testing has failed because it has not installed `sympl`, which should have been installed during `Install dependencies`

I'll fix that, no issues.

**Pull request merged**